docs/dev/apis/lethallib/customscrap.md

```diff
@@ -0,0 +1,94 @@
+- **Item Spawns On Ground**: Should be true for scrap.
+- **Highest Sale Percentage**: Unknown; All items have this set to 80. Set it to that for safety.
+- **Is Conductive Metal**: Whether this item attracts lightning.
+- **Max/Min Value**: The value range for the item. Does not directly correlate to the range in game; a scalar is applied based on the moon it spawns on. For reference, the Big Bolt uses the range Max-80, Min-50.
+- **Spawn Prefab**: The prefab representing the item in the world. Creation of this is detailed later in this guide.
+- **Rotation/Position/Vertical Offset**: The offset values for holding the item in your hand. Mostly needs manual fiddling with until it looks right for a given item.
+- **Grab SFX, Drop SFX, Item Icon**: These must be assigned for the scrap to fully function.
+- **Allow Dropping Ahead of Player**: Whether the item can be dropped in front of the player, as opposed to only dropping directly below their feet.
+- **Is Defensive Weapon**: For scrap that also acts as weapons; requires more special scripting, so for most cases should be left off.
+
+Everything else is optional, has no effect on scrap, or shouldn't be modified (e.g. Netcode variables should be left alone).
+
+## Item Prefab
+An item prefab needs to have a specific setup to work correctly with all the Lethal Company systems. The prefab should have only the root object and a child object called "ScanNode":
+![Image of the Big Bolt scrap prefab structure.](/images/lethallib/customscrap/ScrapPrefab.png)
+
+### Root Object
+The root object should have the following:
+- Mesh Renderer + Mesh Filter for your model
+- Box collider roughly encompassing your model
+- Physics Prop script (see below for configuration)
+- Audio Source with no clip assigned (see below for output setup)
+- Network Object (with default checkbox configuration)
+The root object will also need to have the tag "PhysicsProp" and the layer set to "Props".
+
+The Physics Prop script component should have:
+- The checkboxes for "Grabbable", "Is In Factory", and *optionally* "Grabbable to Enemies" set to true
+- The "Item Properties" assigned to the Item Data you created
+
+The Audio source should use the "Diagetic" (Yes it's misspelled in the game files) audio mixing. If you're using a decompilation you can use the one already present, otherwise you should create an Audio Mixer use the following screenshots as a guide for how to set up the master controller:
+![Image of the audio mixer for diegetic audio.](/images/lethallib/customscrap/AudioMixMaster.png)
+
+### ScanNode Object
+On the ScanNode child gameobject, you should have the following:
+- Box collider tightly encompassing your model
+- Scan Node script
+
+The Scan Node script compontnet should have:
+- Max/Min range set to the max/min distance to have your scrap detectable (13, 1 are typical values)
```

Comment:
I don't know how the scan node component is set up, but if it's Min then Max, change it to that order - otherwise if it's the same as shown here, leave as is